src/test/java/bio/terra/common/fixtures/JsonLoader.java

```diff
@@ -16,12 +16,12 @@
 
 @Component
 public class JsonLoader {
-  private ClassLoader classLoader;
-  private ObjectMapper objectMapper;
+  private final ClassLoader classLoader;
+  private final ObjectMapper objectMapper;
 
   @Autowired
   public JsonLoader(ObjectMapper objectMapper) {
-    this.classLoader = getClass().getClassLoader();
+    classLoader = getClass().getClassLoader();
     this.objectMapper = objectMapper;
```

Comment:
It seems like these two lines don't have the same style. Is this expected?

Reply:
My preference is to only use `this.` when required. That is, when you need to refer to a member that's shadowed by another name in scope. In this case `objectMapper` is a constructor parameter so it shadows the member.

Typically this only happens in constructors or setters but there are other cases where `this` can show up.